src/share/util/shr_const_mod.F90

```diff
@@ -57,10 +57,11 @@ MODULE shr_const_mod
    real(R8),parameter :: SHR_CONST_OCN_REF_SAL = 34.7_R8     ! ocn ref salinity (psu)
    real(R8),parameter :: SHR_CONST_ICE_REF_SAL =  4.0_R8     ! ice ref salinity (psu)
 
-   real(R8),parameter :: SHR_CONST_SPVAL      = 1.0e30_R8     ! special missing value
-   real(R8),parameter :: SHR_CONST_SPVAL_TOLMIN = 0.99_R8 * SHR_CONST_SPVAL  ! min spval tolerance
-   real(R8),parameter :: SHR_CONST_SPVAL_TOLMAX = 1.01_R8 * SHR_CONST_SPVAL  ! max spval tolerance
-
+   real(R8),parameter :: SHR_CONST_SPVAL        = 1.0e30_R8                 ! special missing value
+   real(R8),parameter :: SHR_CONST_SPVAL_TOLMIN = 0.99_R8 * SHR_CONST_SPVAL ! min spval tolerance
+   real(R8),parameter :: SHR_CONST_SPVAL_TOLMAX = 1.01_R8 * SHR_CONST_SPVAL ! max spval tolerance
+   real(R8),parameter :: SHR_CONST_SPVAL_AERODEP= 1.e29_r8                  ! special aerosol deposition
```

Comment:
Is this still needed?

Reply:
No - it is not. But I did not want to remove it in case ACME for some reason used it in its prognostic components. @rljacob - can you please verify if this is needed anywhere on the AMCE side? We can then remove it in a subsequent PR - since I am going to be putting in another cleanup on the dice end.

Reply:
It looks like this is a variable that you newly added in this PR, which did not exist before. So presumably ACME/E3SM cannot be using it right now?

Reply:
I don't think this question has been addressed.

Reply:
No - this is no longer needed. I can remove it in an upcoming PR for dice that I am planning.

Reply:
I see now.  The values used to be called "aerodep_spval" in datm_comp_mod.F90.  Now its no longer used but was still moved to shr_const_mod.F90.  Yes get rid of it if now no one needs it.